internal/adapters/runtime/kubernetes/game_room_watcher.go

```diff
@@ -224,6 +239,9 @@ func (k *kubernetes) WatchGameRoomInstances(ctx context.Context, scheduler *enti
 		k.clientSet,
 		defaultResyncTime,
 		informers.WithNamespace(scheduler.Name),
+		informers.WithTweakListOptions(func(options *metav1.ListOptions) {
+			options.LabelSelector = fmt.Sprintf("%s=%s", maestroLabelKey, maestroLabelValue)
+		}),
```

Comment:
Without this, maestro will receive events from every pod in the namespace, event if those that are not game rooms.